src/CONST.js

```diff
@@ -443,7 +452,9 @@ const CONST = {
         METHOD: {
             POST: 'post',
         },
-        MAX_REQUEST_RETRIES: 10,
+        MIN_RETRY_WAIT_TIME: 10,
```

Comment:
So retry will happen between 10ms and 10s?

Reply:
Yes. On the first retry the wait time will be set to a random value between 10ms and 100ms. With each subsequent retry it will double until it hits a maximum wait time of 10s.

As seen here and as described in the issue and [design doc](https://docs.google.com/document/d/15Cn21gjcMztiirOygIVfNB5uSo9OVOAqHP7RXdsHpNg/edit#heading=h.mp5ur4729g6). Does that sound good?
https://github.com/Expensify/App/blob/e947ba221178de03d50d88348291c602af549272/src/libs/RequestThrottle.js#L10-L20

Reply:
Ok sounds good, I'll make that update. Originally I didn't include MS because I didn't see it in most other cases, but I agree that it's probably best to include the unit.